client/state/jetpack-connect/reducer/jetpack-connect-authorize.js

```diff
@@ -26,28 +26,17 @@ import {
 import { isStale } from '../utils';
 import { JETPACK_CONNECT_AUTHORIZE_TTL } from '../constants';
 
-function buildDefaultAuthorizeState() {
-	return {
-		queryObject: {},
-		isAuthorizing: false,
-		authorizeSuccess: false,
-		authorizeError: false,
-		timestamp: Date.now(),
-		userAlreadyConnected: false,
-		autoAuthorize: false,
-	};
-}
```

Comment:
This was used in one place. A literal object expression gives us a new object anyways. Inline it!